README.md

```diff
@@ -74,27 +74,32 @@ sudo emerge net-misc/curl media-libs/freeglut \
 On Windows:
 
 Download Python for Windows [here](https://www.python.org/downloads/release/python-2711/). This is
-required for the SpiderMonkey build on Windows.
+required for the SpiderMonkey build on Windows. **Note**: even if you enable the "Add python.exe to Path", it's not certain that MSYS2 will pick this up. You may have to add `/c/Python27` and `/c/Python27/Scripts` manually to the path to get things working (in `~/.bash_profile` or similar), once MSYS2 is installed.
 
 Install MSYS2 from [here](https://msys2.github.io/). After you have done so, open an MSYS shell
 window and update the core libraries and install new packages:
 
 ```sh
-update-core
+pacman -Su
 pacman -Sy git mingw-w64-x86_64-toolchain mingw-w64-x86_64-freetype \
```

Comment:
[Partial upgrades are unsupported by pacman](https://wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported) (at least natively on Arch, not sure about on MSYS). We should combine these two lines into `pacman -Syu git mingw-w64-x86_64-toolchain ...`.